qemu/tests/block_stream_check_backingfile.py

```diff
@@ -36,6 +36,13 @@ def check_backingfile(self):
                    "no backing file")
             raise error.TestFail(msg)
 
+    def check_backingfile_exist(self):
+        backingfile = self.get_backingfile()
```

Comment:
@pezhang , qemu is very strange, it will return an absolute path even I give it a relative path (which should be acceptable even though), while self.backingfile is format to relative path, so if use it here, will get wrong result. So I can only use self.base_image, which is absolutely the absolute path of the backing file in this test case.